R/RLearner_regr_glmboost.R

```diff
@@ -22,14 +26,21 @@ makeRLearner.regr.glmboost = function() {
   )
 }
 #' @export
-trainLearner.regr.glmboost = function(.learner, .task, .subset, .weights = NULL, mstop, nu, m, risk, trace, stopintern, ...) {
+trainLearner.regr.glmboost = function(.learner, .task, .subset, .weights = NULL, mstop, nu, m, risk, trace, stopintern, family, custom.family, ...) {
   ctrl = learnerArgsToControl(mboost::boost_control, mstop, nu, risk, trace, stopintern)
   d = getTaskData(.task, .subset)
   f = getTaskFormula(.task)
+  if (!is.null(getHyperPars(.learner)$custom.family)) {
+    family = custom.family
+  } else {
+    family = family
```

Comment:
I dont really like that we overwrite the family automatically and dont give a message when costum family is used. Can we add a dummy value for family that is required to be set to use a costum family?